### PR TITLE
libevdev: update to 1.10.0.

### PR DESCRIPTION
**Pull request by skmpz**

### CODE REVIEW

**Comment by ericonr** [comment]
This should be good to merge, I will wait until we have https://github.com/void-linux/void-packages/pull/27145 to do it, ok?

Please ping me in a while, if I forget.

**Comment by biopsin** [comment]
ping

**Comment by ericonr** [comment]
Since this uses a bunch of kernel header stuff, I will wait for GCC to be merged.

File: srcpkgs/libevdev/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'libevdev'
 pkgname=libevdev
-version=1.9.1
+version=1.10.0
```
Comment (ericonr) [inline srcpkgs/libevdev/template:3]: Hm, this version (or the more recent 1.10.1) aren't in https://www.freedesktop.org/wiki/Software/libevdev/

Weird.

Comment (ericonr) [reply srcpkgs/libevdev/template:3]: Got upstream to fix :)

**Pull request closed without merging**